SwiftUIDZ/SwiftUIDZ/Goods/GoodsView.swift

```diff
@@ -0,0 +1,218 @@
+        ZStack {
+            GradientNavigationView()
+                .frame(width: 350, height: 50)
+                .cornerRadius(10)
+                .shadow(color: .black.opacity(0.4), radius: 10)
+            Text("\(Constants.totalAmountText) \(goodsViewModel.summ)\(Constants.dollarSymbol)")
+                .font(.title2)
+                .bold()
+                .foregroundColor(Color.white)
+        }
+    }
+    
+    private var productsView: some View {
+        VStack {
+            ForEach(0..<goodsViewModel.furnituresCount) { index in
+                makeProductView(index: index)
+            }
+            .padding(.all, 10)
+        }
+    }
+    
+    private var productRectangleView: some View {
+        RoundedRectangle(cornerRadius: 20)
+            .frame(width: 350, height: 110)
+            .foregroundColor(.white)
+            .shadow(color: .black.opacity(0.2), radius: 10)
+    }
+    
+    private var gradientNavigationView: some View {
+        Rectangle()
+            .fill(Color.clear)
+            .frame(height: 70)
+            .background(LinearGradient(
+                colors: [.yellow, Color(Constants.gradientColorTwoName)],
+                startPoint: .topLeading,
+                endPoint: .bottomTrailing)
+            )
+            .ignoresSafeArea(.all)
+    }
+    
+    private var searchBar: some View {
+        HStack {
+            HStack {
+                Image(systemName: Constants.magnifyingglassSystemImageName)
+                TextField(Constants.placeholderTextfieldText, text: $goodsViewModel.searchText, onEditingChanged: { isEditing in
+                }, onCommit: {
+                })
+                .foregroundColor(.primary)
+                Button(action: {
+                    goodsViewModel.searchText = Constants.emptyString
+                }) {
+                    Image(systemName: Constants.xmarkCircleSystemImageName).opacity(goodsViewModel.searchText == Constants.emptyString ? 0 : 1)
+                }
+            }
+            .padding(EdgeInsets(top: 12, leading: 6, bottom: 12, trailing: 6))
+            .foregroundColor(.secondary)
+            .background(Color(.secondarySystemBackground))
+            .cornerRadius(25.0)
+        }
+        .padding(.horizontal, 15)
+    }
+    
+    private var filterButtonView: some View {
+        Button {
+            goodsViewModel.isFilterButtonPressed.toggle()
+        } label: {
+            Image(Constants.filterImageName)
+                .resizable()
+                .frame(width: 30, height: 30)
+        }
+        .padding(.trailing, 20)
+    }
+    
+    // MARK: - Private Methods
+    
+    private func makeProductImageView(index: Int) -> some View {
+        Image(goodsViewModel.getFurnitureImageName(index: index))
+            .resizable()
+            .frame(width: 100, height: 100)
+            .offset(y: -5)
+    }
+    
+    private func makeProductView(index: Int) -> some View {
```

Comment:
```suggestion
    private func makeProductView(index: Int) -> some View {
```
И это -- тоже разнеси